### ISSUE TITLE
Add a `scrolltoUp` button

### ISSUE THREAD

**Issue by afzal442** [meshery/meshery.io#515]
**Current State of the Meshery.io site**

Currently, there is no. It's difficult to scroll from bottom to top.

**Desired State of the Meshery.io site**

We can add a smooth `scrolltoUp` button.

---
### Contributor Resources

The meshery.io website uses Jekyll and GitHub Pages. Site content is found under the [`master` branch](https://github.com/meshery/meshery.io/tree/master).
- See the [Contributing to Meshery.io Website](https://github.com/layer5io/meshery.io#contributing-to-the-mesheryio-website) section of the readme.md and other [contributing instructions](https://docs.meshery.io/project/contributing), too.
- See Meshery site designs in this [Figma designfile](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI?node-id=110%3A1). Fill-in the [Layer5 Community Member Form](https://layer5.io/newcomer) and join the [Layer5 Community Slack](http://slack.layer5.io) for access.

**Comment by harsh9711** [comment]
hi i m interested in your issue

**Comment by harsh9711** [comment]
WOULD YOU LIKE TO SUGGEST ME SOMETHING ANY HINTS OR ANYTHING AS THIS IS MY FIRST CONTRIBUTION

**Comment by warunicorn19** [comment]
Hey @harsh9711 , for meshery.io we use a jekyll framework, maybe you'd like to go through the [documentation](https://jekyllrb.com/) first. Also, please consider using something like 
<img width="110" alt="Screenshot 2021-08-24 at 4 22 51 PM" src="https://user-images.githubusercontent.com/53532328/130604716-0a70ec0c-1946-4c64-9718-046bb104585a.png">
as the scroll to up icon. And if you could please try maintain the colour scheme. Thanks! If you have any questions please join our [slack workspace](https://join.slack.com/t/meshery/shared_invite/zt-rgbyt4kj-6PUUyqY9SfQGRSt_oZV9yQ). Good Luck 😃.

**Comment by harsh9711** [comment]
> Hey @harsh9711 , for meshery.io we use a jekyll framework, maybe you'd like to go through the [documentation](https://jekyllrb.com/) first. Also, please consider using something like
> <img alt="Screenshot 2021-08-24 at 4 22 51 PM" width="110" src="https://user-images.githubusercontent.com/53532328/130604716-0a70ec0c-1946-4c64-9718-046bb104585a.png">
> as the scroll to up icon. And if you could please try maintain the colour scheme. Thanks! If you have any questions please join our [slack workspace](https://join.slack.com/t/meshery/shared_invite/zt-rgbyt4kj-6PUUyqY9SfQGRSt_oZV9yQ). Good Luck .

ok 
please tell about color scheme

**Comment by warunicorn19** [comment]
https://meshery.io/company/brand

**Comment by AnshumanDhiman** [comment]
can I work on this issue @warunicorn19 @afzal442

**Comment by warunicorn19** [comment]
@AnshumanDhiman if you could please confirm with @harsh9711 if he's still working on this. If he isn't then the issue is yours.😃

**Comment by afzal442** [comment]
Thanks, @AnshumanDhiman for your interest. I echo Aditya. 

Even this is a kind of priority. 😃 let's see.

**Comment by AnshumanDhiman** [comment]
@harsh9711 are you still working on this ??

**Comment by afzal442** [comment]
since no updates here, we are assuming @harsh9711 is not working on here. 

Giving it to @AnshumanDhiman! let us know if you need help further.

// @warunicorn19

**Comment by warunicorn19** [comment]
Sounds good!😃

**Comment by AnshumanDhiman** [comment]
Thanks @afzal442 @warunicorn19

**Comment by abhishek818** [comment]
Hello @warunicorn19 @afzal442, can i take up this issue?

**Comment by warunicorn19** [comment]
@AnshumanDhiman i think you have more issues assigned to yourself and considering you also have college, lets have @abhishek818 tackle this, sound good @AnshumanDhiman ?

**Comment by AnshumanDhiman** [comment]
Well I was facing the issue of having working on different branches at the same time that's why I am waiting on one PR to merge or review then work😅.
I was having 1 pr for this can I push actually it was pushing to another issue so I deleted and waiting for other PR to merge to I can work on this

**Comment by AnshumanDhiman** [comment]
https://user-images.githubusercontent.com/67755381/134229970-38ec85e1-d609-4648-928d-33b1bb2b3812.mp4

@warunicorn19 have a look at this. It is in another branch I am not pushing at the same time as it will push it into my recent issue and one thing more can't find any icon to put in the button like fontawesome icon or anything

**Comment by warunicorn19** [comment]
> Hey @harsh9711 , for meshery.io we use a jekyll framework, maybe you'd like to go through the [documentation](https://jekyllrb.com/) first. Also, please consider using something like 
> <img width="110" alt="Screenshot 2021-08-24 at 4 22 51 PM" src="https://user-images.githubusercontent.com/53532328/130604716-0a70ec0c-1946-4c64-9718-046bb104585a.png">
> as the scroll to up icon. And if you could please try maintain the colour scheme. Thanks! If you have any questions please join our [slack workspace](https://join.slack.com/t/meshery/shared_invite/zt-rgbyt4kj-6PUUyqY9SfQGRSt_oZV9yQ). Good Luck 😃.

Aah, okay I understand, we could wait for those PRs to merge, while you're at it could you use this chevron.

**Comment by warunicorn19** [comment]
With our color scheme.
